tweakwcs/linearfit.py

```diff
@@ -542,13 +564,28 @@ def fit_rscale(xy, uv, wxy=None, wuv=None):
     q = np.array([-sthetax, cthetay, yshift], dtype=np.longdouble)
 
     # Return the shift, rotation, and scale changes
-    fit = _build_fit(p, q, fitgeom='rscale')
+    fit = _build_fit(p, q, fitgeom=fitgeom)
     resids = xy - np.dot(uv, fit['matrix_ld'].T) - fit['shift_ld']
     fit['resids'] = resids.astype(np.double)
     _compute_stat(fit, residuals=resids, weights=w)
     return fit
 
 
+def fit_rshift(xy, uv, wxy=None, wuv=None):
+    """ Fits (non-iteratively and without sigma-clipping) a displacement,
+    rotation between input lists of positions ``xy`` and ``uv``. When weights
+    are provided, a weighted fit is performed. Parameter descriptions and
+    return values are identical to those in `iter_linear_fit`, except
+    returned ``fit`` dictionary does not contain the following keys irrelevant
+    to this function: ``'center'``, ``'fitmask'``, and ``'eff_nclip'``.
+
+    .. note::
+        This function is equivalent to calling `fit_rscale` with ``scale=1``.
```

Comment:
Um, if fit_rscale is also fitting a shift, is it a good name?

Reply:
It is OK. It was like this in the `drizzlepac` for a decade and I do not feel making name longer by adding `shifts` to it would help much. This `'general'` fit should probably be `'affine'` or so but no one in the past objected to it and I do not see the need to re-define these now. Definitely not in this PR.